dev-docs/plugins/bc/bc-prebid-plugin-render-options.md

```diff
@@ -38,13 +45,64 @@ When configuring prebid options for more than one ad break, create an array of p
 - [frequencyRules](#frequencyRules)
 - [label](#labeloption)
 
+<a name="adrenderer"></a>
+### adRenderer
+
+**Description:**
+
+This option is used to override the default behavior for selecting the plugin used to render the selected ad(s).
+
+By default, the prebid plugin will select the Brightcove IMA plugin if it detects that DFP is the primary ad server.  If dfpParameters are specified in *any* of the prebid configuration options passed into the plugin, then DFP is identified as the primary ad server.
+
+If the prebid configuration options do not include any dfpParameters, meaning that DFP is *not* the primary ad server, then the MailOnline Prebid plugin will be used to render the ad(s).
+
+Specifying one of the acceptable values listed below will override this behavior; the prebid plugin will use the renderer specified in this option.
+
+{% capture infoNote %}
+The prebid plugin will only use the **_first_** definition of the `adRenderer` option to control the ad renderer selection.  If you define this option more than once, the other specifications will be ignored.
+{% endcapture %}
+
+{% include alerts/alert_note.html content=infoNote %}
+
+{% capture infoNote %}
+If your page includes more than one Brightcove Player within the same HTML document and *any* of these players loads the Brightcove IMA Plugin, then it is recommended that *all* of your players in the document use the Brightcove IMA Plugin. Therefore, even if your prebid configuration does not include DFP parameters and you are not using a "custom" renderer, in this case you should explicitly specify the IMA plugin using the `adRenderer` option as shown below. Doing so seems to prevent a problem observed when multiple players are being used in the same HTML document in PC browsers (Edge and Internet Explorer) and some of the players load the IMA plugin and the others load the MailOnline plugin. You can also prevent this collision if your players are loaded into their own iFrames.
+{% endcapture %}
+
+{% include alerts/alert_note.html content=infoNote %}
+
```

Comment:
Let's hold off on doing this modification to the doc.  We may need to change the wording of it depending on the outcome of our testing here

Reply:
Let's hold off on doing this modification to the doc.  We may need to change the wording of it depending on the outcome of our testing here

Reply:
Let's go ahead and add the infoNote but let's change the wording to the following:

If you will be using the Brightcove IMA plugin as the ad renderer, either because DFP is the primary ad server or you have explicitly specified the IMA plugin in the `adRenderer` option, then it is SUGGESTED that you add the IMA plugin to your Brightcove Player configuration in the Brightcove Studio.  Doing so prevents some problems observed when the Player is running on iOS and the IMA plugin is loaded at run-time.